guides/source/layout.html.erb

```diff
@@ -1,31 +1,37 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title><%= yield(:page_title) %></title>
+  <link rel="preconnect" href="https://rsms.me/">
+  <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
```

Comment:
Should we self-host this instead of making users ping a third party site?

Reply:
We certainly can (https://rsms.me/inter/ is the source site where the link for download/install also lives).

Reply:
Moved locally in 103f5724